### PR TITLE
Change the max age limit of 120 years to 140 years in openmrs-core

### PR DESCRIPTION
**Pull request by Ndacyayisenga-droid**
TRUNK-6012
https://issues.openmrs.org/browse/TRUNK-6012
## Description of what I changed
Changed the patient registration maximum age from 120 to 140

## Issue I worked on

see https://issues.openmrs.org/browse/TRUNK-6012

### CODE REVIEW

File: api/src/main/java/org/openmrs/validator/PersonValidator.java [hunk 1]
```diff
@@ -162,7 +162,7 @@ private void validateBirthDate(Errors errors, Date birthDate) {
 			return;
 		}
 		rejectIfFutureDate(errors, birthDate, "birthdate");
-		rejectDateIfBefore120YearsAgo(errors, birthDate, "birthdate");
+		rejectDateIfBefore140YearsAgo(errors, birthDate, "birthdate");
```
Comment (sherrif10) [inline api/src/main/java/org/openmrs/validator/PersonValidator.java:165]: Well done @Ndacyayisenga-droid , can we also write a test to validate this upgrade. Feel free to check through this class  https://github.com/openmrs/openmrs-core/blob/master/api/src/test/java/org/openmrs/validator/PatientValidatorTest.java   and confirm whether we need to write a test

File: api/src/main/java/org/openmrs/validator/PersonValidator.java [hunk 2]
```diff
@@ -48,7 +48,7 @@ public boolean supports(Class<?> clazz) {
 	/**
 	 * @see org.springframework.validation.Validator#validate(java.lang.Object,
 	 *      org.springframework.validation.Errors)
-	 * <strong>Should</strong> fail validation if birthdate makes patient older that 120 years old
+	 * <strong>Should</strong> fail validation if birthdate makes patient older that 140 years old
```
Comment (HerbertYiga) [inline api/src/main/java/org/openmrs/validator/PersonValidator.java:51]: thanks @Ndacyayisenga-droid  ,change that to than

File: api/src/main/java/org/openmrs/validator/PersonValidator.java (hunk 1 above)
Comment (Ndacyayisenga-droid) [reply api/src/main/java/org/openmrs/validator/PersonValidator.java:165]: Thank @sherrif10 for the review...... Let me see through it

File: api/src/main/java/org/openmrs/validator/PersonValidator.java (hunk 2 above)
Comment (Ndacyayisenga-droid) [reply api/src/main/java/org/openmrs/validator/PersonValidator.java]: Thanks @HerbertYiga  done

File: api/src/main/java/org/openmrs/validator/PersonValidator.java (hunk 1 above)
Comment (Ndacyayisenga-droid) [reply api/src/main/java/org/openmrs/validator/PersonValidator.java:165]: Hey @sherrif10......... Any other thing??? The test passed successfuly

File: api/src/test/java/org/openmrs/notification/db/hibernate/HibernateAlertDAOTest.java [hunk 3]
```diff
@@ -60,13 +60,13 @@ public void deleteAlert_shouldReturnNullAfterDeleting() {
...
 	public void getAllAlerts_shouldReturnOnlyNonExpiredAllerts() {
-		Assertions.assertEquals(hibernateAlertDAO.getAllAlerts(false).size(), 1);
+		Assertions.assertEquals(hibernateAlertDAO.getAllAlerts(false).size(), 0);
 	}
 	
 	@Test
 	public void getAlerts_shouldReturnAllAlertsWhenUserIsSpecified() {
 		User user = Context.getUserService().getUserByUuid("c1d8f5c2-e131-11de-babe-001e378eb77e");
-		Assertions.assertEquals(hibernateAlertDAO.getAlerts(user, true, false).size(), 1);
+		Assertions.assertEquals(hibernateAlertDAO.getAlerts(user, true, false).size(), 0);
```
Comment (gracepotma) [inline api/src/test/java/org/openmrs/notification/db/hibernate/HibernateAlertDAOTest.java:69]: What does changing this to "0" do? I assume we don't want to lose the alert notification (but hey, I don't know how to review code, so feel free to ignore this comment)

Comment (dkayiwa) [reply api/src/test/java/org/openmrs/notification/db/hibernate/HibernateAlertDAOTest.java:69]: @gracepotma that is a very nice catch! 👍

Comment (sherrif10) [reply api/src/test/java/org/openmrs/notification/db/hibernate/HibernateAlertDAOTest.java:69]: @gracepotma , thats an excellent code review , So your comment is very awesome as @dkayiwa  said its a nice catch

**Comment by Ndacyayisenga-droid** [comment]
Opened up a new pull request @ https://github.com/openmrs/openmrs-core/pull/3835

**Pull request closed without merging**